geosnap/_community.py

```diff
@@ -321,6 +322,28 @@ def cluster_spatial(
         comm.models[model_name] = model
         return comm
 
+    def silplot(self, X, labels, **kwargs):
+        """ Returns a silhouette plot of the model that is passed to it.
+
+        Parameters
+        ----------
+        X: ModelResults.X object, required
+        array used to compute plot
+
+        labels: ModelResults.labels object, required
+        outptut labels from model
+
+        kwargs: **kwargs, optional
+              pass through to plot_silhouette()
+
+        Returns
+        -------
+        silhouette plot with given parameters
+
+        """
+        plot = skplt.metrics.plot_silhouette(X, labels, **kwargs)
```

Comment:
then this would be 

`plot = skplt.metrics.plot_silhouette(self.models[model_name][year].X, self.models[model_name][year].labels, **kwargs)`